cob/cob.c

```diff
@@ -1992,41 +2050,41 @@ M0_INTERNAL void m0_cob_tx_credit(struct m0_cob_domain *dom,
 		break;
 	case M0_COB_OP_CREATE:
 		m0_cob_tx_credit(dom, M0_COB_OP_NAME_ADD, accum);
-		TCREDIT(&dom->cd_fileattr_basic, INSERT, FAB, accum);
+		TCREDIT(&dom->cd_fileattr_basic[0], INSERT, FAB, accum);
```

Comment:
Please add comments to say that:
When calculating credit, we can just use the first element in the array.
The result will be the same.